configs/menuAside.js

```diff
@@ -70,6 +70,19 @@ export default [
       },
     ],
   },
+  {
+    label: "Career",
+    icon: mdiFormDropdown,
+    menu: [
+      {
+        label: "All Departments",
+        to:'/careers/allDepartments'
+      },
+      {
+        label: "Dropdown item Dos",
```

Comment:
what is this label used for ?

Reply:
nothing its just demo for further things

Reply:
please remove it from this feature